### PR TITLE
Rework docker image to run as regular user

### PR DESCRIPTION
**Pull request by Santoi**
Adding:
* `Dockerfile` with Python3.10 support and creating regular user.
* `build.sh` and `run.sh` to build and run the container.
Modifying:
* `README.md` with updated instructions to use the Docker container.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    && add-apt-repository ppa:deadsnakes/ppa -y
+RUN apt-get update -y && apt-get install -y \
+        python3.10 
+RUN apt-get update -y && apt-get install -y \
+        python3.10-dev 
+RUN apt-get update -y && apt-get install -y \
+        python3.10-venv 
+RUN apt-get update -y && apt-get install -y \
+        git
+RUN apt-get update && apt-get install ffmpeg libsm6 libxext6  -y
```
Comment (hidmic) [inline docker/Dockerfile]: @Santoi nit: collapse all of these into a single RUN statement.

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
```
Comment (hidmic) [inline docker/Dockerfile]: @Santoi nit: you can get `pip` from debians. You are using a virtualenv anyways, so.

**Review by hidmic — APPROVED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (Santoi) [reply docker/Dockerfile]: Done.

File: docker/Dockerfile (hunk 2 above)
Comment (Santoi) [reply docker/Dockerfile]: Done.

**Pull request merged**